spec/models/page_sequence_spec.rb

```diff
@@ -6,57 +6,65 @@
   let(:claim) { build(:claim) }
 
   describe "#slugs" do
-    it "excludes “current-school” when the claimant still works at the school they are claiming against" do
-      claim.eligibility.employment_status = :claim_school
+    it "only excludes “current-school” when the claimant still works at the school they are claiming against" do
```

Comment:
Minor, and perhaps I'm parsing the sentence wrong but from this description it sounds like where before you were only testing the "if" case, now you're only testing the "only if" case. I'd suggest "excludes “current-school” if and only if the claimant still works at the school they are claiming against". Or two tests.

Reply:
Done.